tests/test_aria_s1_gunw.py

```diff
@@ -194,3 +196,51 @@ def test_gunw_dates_match():
         date(2024, 5, 27),
         date(2024, 5, 3),
     )
+
+
+@unittest.mock.patch('asf_search.search')
+def test_acquisition_frame_coverage(search_mock, acquisition_geojson):
+    results = []
+    for geojson in acquisition_geojson:
+        product = unittest.mock.MagicMock()
+        product.geojson.return_value = geojson
+        results.append(product)
+
+    search_mock.return_value = asf.ASFSearchResults(results)
+    aq = aria_s1_gunw.get_acquisition(25502, date=date(year=2022, month=2, day=12))
```

Comment:
Would it be better to just create a `Sentinel1Acquisition` object directly rather than call `aria_s1_gunw.get_acquisition`, to make it clear we're only testing the `frame_coverage` property?